import/controls/permissionfilterproxymodel.cpp

```diff
@@ -36,8 +38,9 @@ void PermissionFilterProxyModel::setCapability(int currentIndex, int capability)
     QModelIndex sourceIndex = mapToSource(proxyIndex);
```

Comment:
And finally here also (use of `PermissionManager::Capability` instead of `int`).